pyicloud/base.py

```diff
@@ -128,6 +128,12 @@ def _raise_error(self, code, reason):
                 reason + ".  Please wait a few minutes then try again."
                 "The remote servers might be trying to throttle requests."
             )
+        if str(code) == "410":
+            # Gone (410) error maybe occurs after download url of PhotoAsset expires.
+            reason = (
+                reason
+                + ". Please fetch Object(e.g. PhotoAsset) again to refresh download url."
```

Comment:
I think the message should be more generic.

Like "Please fetch the service again.".